src/olympia/devhub/tests/test_views_edit.py

```diff
@@ -38,56 +38,67 @@ def get_section_url(addon, section, edit=False):
 
 
 @override_settings(MEDIA_ROOT=None)  # Make it overridable.
-class TestEdit(TestCase):
+class EditTest(TestCase):
```

Comment:
IIRC It's not just a convention : nose test runner would not pick classes that do not begin with `Test`, so yes, making base classes names not start with `Test` makes sense. I don't disagree with that part, that was a good move :)

My (minor, feel free to ignore it) issue is that, at first glance, it's not enough, you can't easily tell it's a base class: it's inheriting from `TestCase`, and it has `Test` in the name. So I like to add a `Base` suffix or something to indicate that it's just a base class.